packages/calling-component-bindings/src/handlers/createCommonHandlers.ts

```diff
@@ -774,7 +774,7 @@ export const createDefaultCommonCallingHandlers = memoizeOne(
 
       const mainVideoStream = togetherModeStreams.mainVideoStream;
       if (mainVideoStream && mainVideoStream.isAvailable && !mainVideoStream.view) {
-        const createViewResult = await callClient.createView(call.id, mainVideoStream, options);
+        const createViewResult = await callClient.createView(call.id, undefined, mainVideoStream, options);
```

Comment:
For this createView we do have overloaded functions. You probably don't need to undefined here:
![image](https://github.com/user-attachments/assets/599c7e05-0bf4-40e9-a4db-7e3df26fa1ff)